prac_09/taxi.py

```diff
@@ -0,0 +1,36 @@
+"""
+CP1404/CP5632 Practical
+Car class
+"""
+from prac_09.car import Car
+
+
+class Taxi(Car):
+    """Specialised version of a Car that includes fare costs."""
+    price_per_km = 1.23
+
+    def __init__(self, name, fuel):
+        """Initialise a Taxi instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        # self.price_per_km = price_per_km
+        self.current_fare_distance = 0
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return f"{super().__str__()}, {self.current_fare_distance}km on current fare, ${self.price_per_km:.2f}/km"
+
+    def get_fare(self):
+        """Return the price for the taxi trip, with added rounding."""
+        fare = self.price_per_km * self.current_fare_distance
+        rounded_fare = round(fare, 1)
```

Comment:
This seems to combine general (fare) and specific (rounded_fare) names. You could avoid needing an extra variable name by putting everything on the same line: `fare = round(calculation, 1)`